lambda-deployment-orca/lambda-deployment-orca.gradle

```diff
@@ -24,38 +24,39 @@ spinnakerPlugin {
-  compileOnly (group: 'com.netflix.spinnaker.orca', name: 'orca-core', version: "${orcaVersion}")
-  compileOnly ("org.projectlombok:lombok:1.18.12")
-  compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-artifacts', version: "${korkVersion}")
-  compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-plugins-spring-api', version: "${korkVersion}")
-  implementation("com.amazonaws:aws-java-sdk:1.11.877")
-  implementation("org.apache.commons:commons-lang3:3.8.1")
-  implementation("com.github.ben-manes.caffeine:guava:2.8.5")
-  implementation("com.netflix.awsobjectmapper:awsobjectmapper:1.11.877")
-  implementation("org.apache.httpcomponents:httpclient:4.5.9")
-  implementation("org.apache.httpcomponents:httpcore:4.4.11")
-  implementation("com.squareup.okhttp3:okhttp:4.2.2")
-  implementation("com.squareup.okhttp3:okhttp-apache:3.13.1")
-  annotationProcessor("org.pf4j:pf4j:3.2.0")
-  annotationProcessor ("org.projectlombok:lombok:1.18.12")
-  testImplementation (group: 'com.netflix.spinnaker.orca', name: 'orca-api', version: "${orcaVersion}")
-
-  testImplementation "org.junit.jupiter:junit-jupiter-api:5.5.2"
-  testImplementation group: 'io.strikt', name: 'strikt-core', version: '0.22.1'
-  testImplementation group: 'dev.minutest', name: 'minutest', version: '1.10.0'
-  testImplementation group: 'io.mockk', name: 'mockk', version: '1.9.3'
-  testCompile "com.github.tomakehurst:wiremock-jre8:2.29.1"
-  testImplementation "ru.lanwen.wiremock:wiremock-junit5:1.3.1"
-  testCompile group: 'org.mockito', name: 'mockito-core', version: '2.19.0'
-  testCompile group: 'org.mockito', name: 'mockito-junit-jupiter', version: '2.19.0'
-  testCompile (group: 'com.netflix.spinnaker.orca', name: 'orca-clouddriver', version: "${orcaVersion}")
-
-  testRuntime "org.junit.jupiter:junit-jupiter-engine:5.4.0"
-  testRuntime "org.junit.platform:junit-platform-launcher:1.4.0"
-  testRuntime "org.junit.platform:junit-platform-commons:1.5.2"
+  implementation(enforcedPlatform("io.spinnaker.orca:orca-bom:${orcaVersion}"))
+  implementation("com.amazonaws:aws-java-sdk")
+  implementation("com.netflix.awsobjectmapper:awsobjectmapper")
+  implementation("org.apache.commons:commons-lang3")
+  implementation("com.squareup.okhttp3:okhttp")
+
+  compileOnly(group: 'org.pf4j', name: 'pf4j')
+  compileOnly (group: 'io.spinnaker.orca', name: 'orca-api')
+  compileOnly (group: 'io.spinnaker.orca', name: 'orca-clouddriver')
+  compileOnly (group: 'io.spinnaker.orca', name: 'orca-core')
+  compileOnly ("org.projectlombok:lombok")
+  compileOnly (group: 'io.spinnaker.kork', name: 'kork-artifacts')
+  compileOnly (group: 'io.spinnaker.kork', name: 'kork-plugins-spring-api')
+
+  annotationProcessor platform("io.spinnaker.orca:orca-bom:${orcaVersion}")
+  annotationProcessor ("org.projectlombok:lombok")
+  annotationProcessor("org.pf4j:pf4j")
+
+  testImplementation (platform("io.spinnaker.orca:orca-bom:${orcaVersion}"))
```

Comment:
All these test dependencies aren't really required for this PR right?  Plus also, with
```
implementation(enforcedPlatform("io.spinnaker.orca:orca-bom:${orcaVersion}"))
```
there's no need to specify it again as a `testImplementation`.

Reply:
yeah i added `testImplementation platform` to be descriptive i can remove it

About the test dependencies just got rid of the version tag and changed the sintax to `group: '', name:''`
i didn’t remove or added any dependencies

Reply:
Aaah, ok.